api/libproject/devstart.rb

```diff
@@ -598,6 +643,13 @@ def run
   :fn => lambda { |*args| create_db_creds(*args) }
 })
 
+Common.register_command({
+  :invocation => "update-user-registered-status",
+  :description => "Adds or removes a specified user from the registered access domain.\n" \
+                  "Accepts three flags: --action [add/remove], --account [admin email], and --user [target user email]",
```

Comment:
I would leave these to the `--help` option. Otherwise, one might forget to update this text when updating the actual options above.